recipes/json-schema-validator/all/conanfile.py

```diff
@@ -47,14 +42,14 @@ def configure(self):
             del self.options.fPIC
 
     def requirements(self):
-        self.requires("nlohmann_json/3.10.5")
+        self.requires("nlohmann_json/3.11.2")
 
     def validate(self):
-        version = tools.Version(self.version)
+        version = scm.Version(self.version)
         min_vs_version = "16" if version < "2.1.0" else "14"
         min_cppstd = "17" if self.settings.compiler == "Visual Studio" and version < "2.1.0" else "11"
         if self.settings.get_safe("compiler.cppstd"):
```

Comment:
```suggestion
        if self.info.settings.get_safe("compiler.cppstd"):
```
https://docs.conan.io/en/latest/migrating_to_2.0/recipes.html#the-validate-method